ups/pipe_tasks.table

```diff
@@ -1,10 +1,10 @@
+setupRequired(afw)
```

Comment:
I suggest making meas_base required, as well, since it is used by mocks.

Also, while you are in here, please add `daf_persistence` as required (used in mocks/simpleMapper.py)